### DISCUSSION TITLE
Error dalam memisahkan koordinat pada data shp

### DISCUSSION THREAD

**Discussion by Argo2226** [teamalgoritma/community#723] [Q&A] [answered]
Selamat pagi, izin bertanya. Saya ingin menggunakan map pada projek rstudio saya, namun ketika saya memisahkan koordinat pada data shp terdapat error seperti berikut. Mohon bantuan untuk solusinya. Terima kasih banyak 

Struktur data
![image](https://github.com/teamalgoritma/community/assets/75960751/ee53a726-2bb0-410f-9d16-093cca1bf39c)

Code dan error
![image](https://github.com/teamalgoritma/community/assets/75960751/757c51c9-eec5-4cb7-b6cc-056f1f3a50f0)

**Comment by ramdhinafinita** [comment, marked as answer]
Dear Pak @Argo2226 

Sebelumnya, pemisahan koordinat pada data shp yang bapak maksud untuk apa ya? karena jika sudah di lakukan select hanya kolom provinsi saja, kolom yang digunakan untuk memisah koordinat tidak ada. Atau Bapak bisa menyertakan file yang digunakan dan code lengkapnya agar kami bisa track dengan mudah.  Bisa dicoba untuk menggunakan data  geojson juga, berikut referensinnya https://algotech.netlify.app/blog/creating-choropleth-with-mapshaper-and-r/
Terimakasih

**Reply by kevbow** [reply]
Halo @Argo2226, untuk menjaga agar forum diskusi tetap berjalan kondusif, kami izin melakukan lock/close diskusi ini. Semoga dapat dipahami.
